internal/verification/reference.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package verification
+
+import (
+	"fmt"
+
+	"github.com/pelletier/go-toml"
+)
+
+// ReferenceValues given by the product team to verify provenances against.
+type ReferenceValues struct {
+	// The digests of the binaries whose provenance the product team wants to verify.
+	BinarySHA256Digests []string `toml:"binary_sha256_digests"`
```

Comment:
Not sure why these are only sha256? Could they not be arbitrary digests, as long as they have the appropriate prefix?